x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/optimizer/LogicalVerifier.java

```diff
@@ -47,6 +50,12 @@ public Failures verify(LogicalPlan plan, boolean skipRemoteEnrichVerification) {
             }
         });
 
+        if (datatypeEquals(planBefore.output(), planAfter.output()) == false) {
+            failures.add(
+                fail(planAfter, "Layout has changed from [{}] to [{}]. ", planBefore.output().toString(), planAfter.output().toString())
```

Comment:
```suggestion
                fail(planAfter, "Output has changed after optimization from [{}] to [{}]. ", planBefore.output().toString(), planAfter.output().toString())
```
"Layout" is a very specific term that only really exists for physical operators.

Reply:
done